internal/pkg/cli/deploy.go

```diff
@@ -183,6 +253,13 @@ func BuildDeployCmd() *cobra.Command {
 			if err != nil {
 				return err
 			}
+
+			if cmd.Flags().Changed(yesInitWorkloadFlag) {
+				opts.yesInitWkld = aws.Bool(false)
+				if initWorkload {
+					opts.yesInitWkld = aws.Bool(true)
+				}
+			}
```

Comment:
why this? instead of just setting `vars.yesInitWkld` down below??

Reply:
ah i think sort of answered this below, but +1 to @iamhopaul123's question there - it seems like this is another way of doing the same thing?

Reply:
I hadn't seen this way of using a pointer, but using this construct instead of two mutually exclusive flags actually simplifies the logic in a pleasant way. This way the behavior is clear and the user is only prompted if they don't specify the flag one way or another via `--init-wkld=false` or `--init-wkld`.